rules/windows/process_creation/proc_creation_win_susp_browser_launch_from_document_reader_process.yml

```
@@ -31,7 +32,12 @@ detection:
             - '\vivaldi.exe'
             - ''
```

```suggestion
            - ''
```
This one has to be a mistake, hasn't it? That is a trivial `true`.